### ISSUE TITLE
False alarm on lexical illusions

### ISSUE THREAD

**Issue by suchow** [amperser/proselint#1191]
Test document:

```
the theory of relativity
```

Output:

```
(base) ➜  Downloads proselint issue.md
issue.md:1:2: lexical_illusions.misc There's a lexical illusion here: a word is repeated.
```

[issue.md](https://github.com/amperser/proselint/files/6766624/issue.md)

**Comment by suchow** [comment]
Here are a bunch of examples:

- `The theory` (the the)
- `the practitioner's side` (s s)
- `an angle` (an an)
- `Applying Newton's second law` (s s)
- `an analytical solution` (an an)
